draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -709,7 +709,7 @@ the token and nonce of the MP_JOIN for which authentication shall be performed.
 
 The MP_RTT option is used to transmit RTT values in milliseconds and
 MUST belong to the path over which this information is transmitted.
-Additionally, the age of the measurement is specified in milliseconds.
+Additionally, the age of the last measurement is specified in milliseconds.
 
 The RTT and Age information is a 32-bit integer, which allows to cover a period of
```

Comment:
Reword:

"The RTT and Age information are 32-bit integers, providing a range of approximately1193 hours"

Reply:
see https://github.com/markusa/ietf-multipath-dccp/pull/81/commits/5ee9a808cfba4bb9d0c935019c2bfab3184169f8